src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -60,15 +84,68 @@ export class CourseExercisesComponent implements OnInit, OnDestroy {
         this.paramSubscription.unsubscribe();
     }
 
-    public groupExercises(selectedOrder: number): void {
+    get filterType() {
+        return ExerciseFilter;
+    }
+
+    /**
+     * Return the total number of exercises for the logged in user
+     */
+    getNrOfExercises(): number {
+        return sum([...this.exerciseCountMap.values()]);
```

Comment:
Maybe use `Array.from(this.exerciseCountMap.values())`? But TBH I am not sure if this works.

Reply:
It works, I changed it